services/dolly/docker-compose-example.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  dolly:
+    build:
+      context: ../dolly
+    restart: unless-stopped
+    environment:
+      type: 'box'
+      box_sub_type: 'enterprise'
+      token: 'example-token'
+      override: '0'
+      bwlimit: '500k'
+      numcheckers: '2'
+      clvl: '0'
+    volumes:
+      - /data/log:/data:rw
```

Comment:
Should be a volume mount? Or at least show that as an example as we should likely be doing it that way anyway.

Reply:
Can we reduce to just `:r`?

Reply:
I thought we are already mounting `/data/log` to `/data` within the container? Not too familiar with docker so I can be totally wrong.

Reply:
The test deploy maps a volume to `/data` but we could also map a local FS path to /data. Either way, does dolly need `rw` access to /data to work? Seems like dolly should only  be reading from that path.